backend/backend/urls.py

```diff
@@ -39,6 +40,7 @@
                     path("", include((organization_router.urls, "organization"))),
                     path("", include((user_router.urls, "user"))),  # for secure jwt authent
                     path("api-token-auth/", obtain_auth_token),  # for expiry token authent
+                    path("api-token-tap/", obtain_auth_token_already_authenticated),
```

Comment:
The plan is to eventually replace all `api-token-auth/` usage with this new endpoint. People will get a token in the frontend and use that in their scripts rather than use their uname/passw in `Client.login`.